src/Microsoft.DotNet.Build.Tasks/PackageFiles/scripts/vstsagent/cleanupagent.proj

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Clean">
+  <PropertyGroup>
+    <BuildToolsTargets45 Condition="'$(OS)' == 'Windows_NT'">true</BuildToolsTargets45>
+  </PropertyGroup>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Build.Common.props))\Build.Common.props" />
+  <UsingTask TaskName="CleanupVSTSAgent" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll" />
+
+  <PropertyGroup>
+    <RetentionDays Condition="'$(RetentionDays)' == ''">5</RetentionDays>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <SourceFolderJsons Include="$(AgentDirectory)/_work/SourceRootMapping/**/SourceFolder.json" />
```

Comment:
You're right,  I refactored it into the task because I thought it made more sense there,  but I neglected to clean this up here